components/delegations/cells/DelegateCell.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import AddressDisplay from 'components/address/AddressDisplay';
+import type { Delegation } from 'lib/delegate/DelegatePlatform';
+
+interface Props {
+  delegation: Delegation;
+}
+
+const DelegateCell = ({ delegation }: Props) => {
+  return (
```

Comment:
I recommend returning `AddressCell` here so that it also includes the link / copy button automatically.